strongbox-storage/strongbox-storage-core/src/main/java/org/carlspring/strongbox/configuration/MutableProxyConfiguration.java

```diff
@@ -144,7 +144,6 @@ public String toString()
                           .add("host", host)
                           .add("port", port)
                           .add("username", username)
-                          .add("password", password)
```

Comment:
This should not be removed, but rather displayed as redacted:
```
    .add("password", "[credentials redacted]")
```

Reply:
Excuse me for joining in late, but two things:
1) If the password is null, append `null`. (This should never be null, BUT it might as well be the case that during some test, someone might have overlooked it. Having it printed out in the logs as `null` should not be a security concern, in my opinion).
2) If the passwrod is set, then I think the better option would be to mask it with `********`. (People will know it's masked/redacted, for sure).